core/language/src/main/java/datawave/core/query/language/builder/jexl/FieldQueryNodeBuilder.java

```diff
@@ -16,26 +29,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import java.util.ArrayList;
-import java.util.List;
-
-import org.apache.lucene.queryparser.flexible.core.QueryNodeException;
-import org.apache.lucene.queryparser.flexible.core.builders.QueryBuilder;
-import org.apache.lucene.queryparser.flexible.core.nodes.EscapedNodes;
-import org.apache.lucene.queryparser.flexible.core.nodes.FieldQueryNode;
-import org.apache.lucene.queryparser.flexible.core.nodes.FuzzyQueryNode;
-import org.apache.lucene.queryparser.flexible.core.nodes.QueryNode;
-import org.apache.lucene.queryparser.flexible.core.nodes.QuotedFieldQueryNode;
-import org.apache.lucene.queryparser.flexible.core.util.UnescapedCharSequence;
-import org.apache.lucene.search.TermQuery;
-
 import com.google.common.collect.Sets;
```

Comment:
Should we put all the imports above the license comment?

Reply:
i have no idea.  that's just the import sorting plugin at work.

Reply:
also, these are all deprecated.  that being said... as you wish.